release-notes.txt

```diff
@@ -1,3 +1,296 @@
+RIOT-2015.09 - Release Notes
+============================
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things: 8-bit
```

Comment:
low-end devices that are typically found in the Internet of Things: devices based on 8-bit